_integration-schemas/klaviyo/lists.md

```diff
@@ -0,0 +1,44 @@
+---
+tap: "klaviyo"
+version: "1.0"
+
+name: "lists"
+doc-link: ""
+singer-schema: "https://github.com/singer-io/tap-klaviyo/blob/master/tap_klaviyo/schemas/lists.json"
+description: ""
+
+replication-method: "Full Table"
+
+api-method:
+    name: ""
+    doc-link: "https://www.klaviyo.com/docs/api/metrics"
```

Comment:
This is a little tricky. The tap repo says it uses the Metrics API (which it does), but not for this endpoint.

If you look [here](https://github.com/singer-io/tap-klaviyo/blob/master/tap_klaviyo/__init__.py#L12), you'll see the endpoint we're using to get the data for the `lists` table:

```python
ENDPOINTS = {
    'global_exclusions': 'https://a.klaviyo.com/api/v1/people/exclusions',
    'lists': 'https://a.klaviyo.com/api/v1/lists',
    'metrics': 'https://a.klaviyo.com/api/v1/metrics',
    'metric': 'https://a.klaviyo.com/api/v1/metric/',
}
```

And in Klaviyo's docs, there's actually a Lists API in the sidenav. Opening that up, we can see in the list of endpoints that there's a [URL that matches the one we're using](https://www.klaviyo.com/docs/api/lists#lists): `https://a.klaviyo.com/api/v1/lists`

That would make this section look like this:

```
api-method:
  name: "Lists in Account"
  doc-link: "https://www.klaviyo.com/docs/api/lists#lists"
```